_sass/KSEAWeb.scss

```diff
@@ -3,4 +3,56 @@
 
 body {
   font-family: "NanumSquareRound";
+  color: #000000; // Default Text Color: Black
+
+  .container {
+    width: 1182px;
+    max-width: 100%;
+    margin: 0 auto;
+  }
+}
+
+header {
```

Comment:
@Drag-yong 
Do we need to separate header SCSS implementation into another file?